#! /bin/sh /usr/share/dpatch/dpatch-run
## 02_configure.dpatch by  <Michele Baldessari <michele@pupazzo.org>>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: On Debian there is no nss3 and nspr4 include path

@DPATCH@
diff -urNad freeipa-1.2.0~/ipa-server/configure.ac freeipa-1.2.0/ipa-server/configure.ac
--- freeipa-1.2.0~/ipa-server/configure.ac	2008-11-14 15:55:28.000000000 +0100
+++ freeipa-1.2.0/ipa-server/configure.ac	2008-11-24 22:30:11.000000000 +0100
@@ -22,8 +22,8 @@
 dnl ---------------------------------------------------------------------------
 dnl - Check for NSPR
 dnl ---------------------------------------------------------------------------
-AC_CHECK_HEADER(nspr4/nspr.h)
-if test "x$ac_cv_header_nspr4_nspr_h" = "xno" ; then
+AC_CHECK_HEADER(nspr/nspr.h)
+if test "x$ac_cv_header_nspr_nspr_h" = "xno" ; then
 	AC_MSG_ERROR([Required NSPR header not available (nspr-devel)])
 fi
 
@@ -31,10 +31,10 @@
 dnl - Check for NSS
 dnl ---------------------------------------------------------------------------
 SAVE_CPPFLAGS=$CPPFLAGS
-CPPFLAGS="-I/usr/include/nspr4"
-AC_CHECK_HEADER(nss3/nss.h)
+CPPFLAGS="-I/usr/include/nspr"
+AC_CHECK_HEADER(nss/nss.h)
 CPPFLAGS=$SAVE_CPPFLAGS
-if test "x$ac_cv_header_nss3_nss_h" = "xno" ; then
+if test "x$ac_cv_header_nss_nss_h" = "xno" ; then
 	AC_MSG_ERROR([Required NSS header not available (nss-devel)])
 fi
 
@@ -44,7 +44,7 @@
 
 # Need to hack CPPFLAGS to be able to correctly detetct slapi-plugin.h
 SAVE_CPPFLAGS=$CPPFLAGS
-CPPFLAGS="-I/usr/include/nspr4"
+CPPFLAGS="-I/usr/include/nspr"
 AC_CHECK_HEADER(dirsrv/slapi-plugin.h)
 CPPFLAGS=$SAVE_CPPFLAGS
 
@@ -123,7 +123,7 @@
 dnl The mozldap libraries are always needed because ipa-slapi-plugins/dna/ 
 dnl will not build against OpenLDAP.
 SAVE_CPPFLAGS=$CPPFLAGS
-CPPFLAGS="-I/usr/include/nspr4 -I/usr/include/nss3"
+CPPFLAGS="-I/usr/include/nspr -I/usr/include/nss"
 AC_CHECK_HEADER(svrcore.h)
 if test "x$ac_cv_header_svrcore_h" = "xno" ; then
 	AC_MSG_ERROR([Required svrcore header not available (svrcore-devel)])
